### DISCUSSION TITLE
Ressource Planning Hebdo par plage

### DISCUSSION THREAD

**Discussion by JeremieDENAEGEL** [witsa/synapps#765] [Q&A]
Bonjour,

Est-ce-que quelqu'un aurait réussi à reproduire le planning hebdo par plage sur synapps ?

**Comment by Mathis554** [comment]
Bonjour;
Tu trouveras ton bonheur ici #471.

**Reply by JeremieDENAEGEL** [reply]
Je demande pour le planning hebdomadaire par plage qui est différent du planning hebdomadaire normal.
Le composite du planning hebdo ne répond pas à ce besoin

Actuellement, j'utilise un synoptique pour contourner ce besoin mais à des moment le lien url du planning envoyé par argscript n'est pas bien pris en compte par le synoptique et cela me fait un acteur non affiché sur le synoptique.
